doc/api/stream.md

```diff
@@ -1580,6 +1595,12 @@ changes:
   - version: v13.10.0
     pr-url: https://github.com/nodejs/node/pull/31223
     description: Add support for async generators.
+  - version: v14.0.0
+    pr-url: https://github.com/nodejs/node/pull/32158
+    description: The `pipeline(..., cb)` will wait for the `'close'` event
+                 before invoking the callback. The implementation tries to
+                 detect legacy streams and only apply this behavior to streams
+                 which are expected to emit `'close'`.
```

Comment:
> The implementation tries to
>                 detect legacy streams and only apply this behavior to streams
>                 which are expected to emit `'close'`.

Actually that's not part 14.0.0, I'd say it deserves its own item.

```suggestion
                 before invoking the callback.
 - version: 14.1.0
    pr-url: https://github.com/nodejs/node/pull/33058
    description: The `pipeline(..., cb)` tries to detect legacy streams and
                 don't wait for the `'close'` event to invoke the callback.
```

Reply:
@aduh95 It is part of v14. The detection has been improved since 14.0.0 but it was already there.

Reply:
Good to know, sorry for the confusion.